index.html

```diff
@@ -4,14 +4,22 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" type="text/css" href="css/home.css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
     <title>Hostal las paisas</title>
 </head>
 <body>
     <nav>
          <!-- Navigation section -->
     </nav>
-    <section>
+    <section class="homepage_hero">
         <!-- Hero section, the cover for the page -->
+        <div class="homepage_hero__layer"></div>
+        <article class="homepage_hero__article">
+            <h1 id="" class="homepage_hero__headline">Hostal <br/> Las paisas</h1>
```

Comment:
Instead of using a `<br>` you could put a max width to h1 to force the line break

Reply:
I changed the <br/> for two different <h1> because setting a max width will break down the design at this moment. It will force the break but in an unexpected way, breaking h1 in the third word, instead of the second one.